about.html

```diff
@@ -0,0 +1,113 @@
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8"
+    crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/bf66e12569.js" crossorigin="anonymous"></script>
+  <title>About</title>
+</head>
+
+<body>
+  <header class="header">
+    <div id="hide-div"><i class="fa-solid fa-bars menu-bars"></i></div>
+    <div class="mobile-menu">
+      <i class="fa-solid fa-x menu-x"></i>
+      <ul class="menu-mobile-list unic-font">
+        <li class="menu-mobile-list-itens"><a href="about.html" class="no-underscore color-three">About</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three"><a href="index.html" class="no-underscore color-three">Program</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three">Join</li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three">Sponsor</li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three">News</li>
+      </ul>
+    </div>
+    <div class="desktop-menu">
+      <ul class="desktop-list color-three">
+        <li class="desktop-list-itens">English</li>
+        <li class="desktop-list-itens">My Page</li>
+        <li class="desktop-list-itens">Logout</li>
+      </ul>
+      <ul class="menu-desktop-list">
+        <img src="./assets/logo/image.jpg" alt="bgs logo" class="bgs-logo">
+        <li class="menu-desktop-list-itens"><a href="about.html" class="no-underscore color-one">About</a></li>
+        <li class="menu-desktop-list-itens color-one"><a href="index.html" class="no-underscore color-one">Program</a></li>
+        <li class="menu-desktop-list-itens color-one">Join</li>
+        <li class="menu-desktop-list-itens color-one">Sponsor</li>
+        <li class="menu-desktop-list-itens color-one">News</li>
+      </ul>
+    </div>
+    <section class="header-about-section d-flex flex-column align-items-center px-3 py-5">
+      <p class="color-two event-about-call ">"Hello game lovers!"</p>
+      <h1 class="event-about-name">BRASIL GAME SHOW 2022</h1>
+      <p class="event-info">The BGS is an event for those who like everyhing about the geek world. Every year improving
+        the event to be better.</p>
+      <h2 class="event-text">For further informations about the event, enter the link above</h2>
+      <h3 class="event-email"><a href="https://brasilgameshow.zendesk.com/hc/pt-br" class="no-underscore color-one"
+          target="_blank">brasilgameshow.zendesk.com</a></h3>
+    </section>
```

Comment:
- [ ] As the requirement you should use the same color, and font provided by microverse. please follow the requirement. you need to use the Font: Cocogoose, Lato, and color: * Basic color swatch: #272A31, #EC5242, #D3D3D3.